frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -426,10 +427,14 @@ export const funnelLogic = kea<funnelLogicType>({
         ],
         steps: [
             () => [selectors.results, selectors.stepsWithNestedBreakdown, selectors.filters],
-            (results, stepsWithNestedBreakdown, filters): FunnelStepWithNestedBreakdown[] =>
-                !!filters.breakdown
+            (results, stepsWithNestedBreakdown, filters): FunnelStepWithNestedBreakdown[] => {
+                if (!Array.isArray(results)) {
```

Comment:
This was causing a white screen of death for funnel time conversion because it was trying to spread an object of time conversion bin result values. This is a temporary-ish fix, I will take a deeper look in a different PR. Not this one... no more merge conflicts plz 😩